### DISCUSSION TITLE
Inspiration to auto publish collections to Galaxy via GitHub

### DISCUSSION THREAD

**Discussion by shawndwells** [CrowdStrike/ansible_collection_falcon#14] [Ideas] [answered]
@redhatrises what do you think of https://www.jeffgeerling.com/blog/2020/automatically-building-and-publishing-ansible-galaxy-collections ? Is this feasible for us?

**Comment by redhatrises** [comment]


**Reply by shawndwells** [reply]
Yeah, this is old. Closing.

**Comment by redhatrises** [comment, marked as answer]
Auto publishing is enabled and can be viewed at https://github.com/CrowdStrike/ansible_collection_falcon/blob/main/.github/workflows/release.yml

Updates are published to https://galaxy.ansible.com/crowdstrike/falcon
